domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3236,7 +3270,7 @@ public void selectSingle(AsyncResult<SQLConnection> conn, String sql, Handler<As
   }
 
   /**
-   * Run a parameterized/prepared select query with the readonly connection
+   * Run a parameterized/prepared select query
```

Comment:
`Run an SQL statement that may write data and returns data, for example {@code SELECT nextval('foo')} or {@code UPDATE ... RETURNING ...}. The first record is returned, or null if there is no result.`